src/zino/snmp/netsnmpy_backend.py

```diff
@@ -79,6 +79,10 @@ def __init__(self, device: PollDevice):
         )
         self.session.open()
 
+    def __del__(self):
+        """Ensures Net-SNMP resources are properly released when sessions are garbage collected"""
+        self.session.close()
+
```

Comment:
Do we need this commit if we end up replacing it anyway?

Reply:
I made a choice to keep it so that the history will show that this was a dead-end solution (and potentially, maybe, serve as a warning to not try it again ;-) )

Reply:
Good, I approve of that, just wanted to make sure it was intentional